ietf/static/css/document_html_txt.scss

```
@@ -0,0 +1,348 @@
+:root {
+  --line: 1.2em;
+  --block: 0 0 0 3ch;
+  --paragraph: var(--line) 0 var(--line) 3ch;
+}
+
+// WeasyPrint can't handle CSS variables in multi-attribute properties, so work
+// around that
+// https://github.com/Kozea/WeasyPrint/issues/1219
+
+@mixin margin-paragraph {
+  margin-top: var(--line);
+  margin-right: 0;
+  margin-bottom: var(--line);
+  margin-left: 3ch;
+}
+
+@mixin margin-line {
+  margin-top: var(--line);
+  margin-right: 0;
+  margin-bottom: var(--line);
+  margin-left: 0;
+}
+
+@mixin margin-block {
+  margin-top: 0;
+  margin-right: 0;
+  margin-bottom: 0;
+  margin-left: 3ch;
+}
+
+.xml2rfc {
+
+  line-height: var(--line);
+  width: 72ch;
+
+h1, h2, h3, h4, h5, h6 {
+  font-weight: bold;
+  font-size: inherit;
+  line-height: inherit;
+  // margin: var(--line) 0;
+  @include margin-line;
+}
+.section-number {
+  margin-right: 1ch;
+}
+p {
+  // margin: var(--paragraph);
+  @include margin-paragraph;
+}
+aside, ol, dl {
+  // margin: var(--block);
+  @include margin-block;
+}
+figure {
+  margin: 0;
+}
+blockquote {
+  // margin: var(--paragraph);
+  @include margin-paragraph;
+  border-left: 2px solid darkgrey;
+}
+
+/* Header junk */
+#external-metadata {
+  display: none !important; /* metadata.min.js is evil because it produces unstyleable goop */
+}
+#identifiers dt:is(.label-stream, .label-rfc, .label-std, .label-bcp, .label-internet-draft,
+                   .label-workgroup, .label-intended-status, .label-obsoletes, .label-updates,
+                   .label-published, .label-expires, .label-category, .label-issn,
+                   .label-authors), .ears {
+  display: none;
+}
+#identifiers {
+  margin: 0;
+  display: grid;
+  grid-template-columns: 47ch 24ch;
+  column-gap: 1ch;
+  grid-auto-flow: column;
+  grid-auto-rows: auto;
+  row-gap: 0;
+}
+#identifiers dt {
+  margin: 0 1ch 0 0;
+  float: revert;
+  display: inline-block;
+}
+#identifiers dd {
+  margin: 0;
+  width: 47ch;
+}
+#identifiers dd::before {
+  margin: 0 1ch 0 0;
+}
+#identifiers dd.rfc::before {
+  content: "Request for Comments:";
+}
+#identifiers dd.std::before {
+  content: "STD:";
+}
+#identifiers dd.bcp::before {
+  content: "BCP:";
+}
+#identifiers dd.internet-draft::before {
+  content: "Internet-Draft:";
+}
+#identifiers dd.workgroup::before {
+  content: "Workgroup:";
+}
+#identifiers dd.intended-status::before {
+  content: "Intended Status:";
+}
+#identifiers dd.obsoletes::before {
+  content: "Obsoletes:";
+  margin-left: -11ch;
+}
+#identifiers dd.obsoletes {
+  padding-left: 11ch;
+  width: 36ch;
+}
+#identifiers dd.updates::before {
+  content: "Updates:";
+  margin-left: -9ch;
+}
+#identifiers dd.updates {
+  padding-left: 9ch;
+  width: 38ch;
+}
+#identifiers dd.updates::first-line {
+  padding-left: 0;
+}
```

```suggestion
#identifiers dd.obsoletes::before {
  content: "Obsoletes:";
  margin: 0 0 0 -10ch;
}
#identifiers dd.obsoletes {
  padding-left: 10ch;
  width: 37ch;
}
#identifiers dd.updates::before {
  content: "Updates:";
  margin: 0 0 0 -8ch;
}
#identifiers dd.updates {
  padding-left: 8ch;
  width: 39ch;
}
#identifiers dd:is(.updates, .obsoletes) {
  /* HAXX: this gets around the lack of text-content-trim support */
  display: flex;
  flex-wrap: wrap;
}
#identifiers dd:is(.updates, .obsoletes) a {
  margin: 0 0 0 1ch;
}
```

The abomination grows even worse.  Now we have flexbox inside grid for maximum awesomeness.  I am working around a "bug" in xml2rfc here with this change.